core-application/src/test/java/gohigher/application/service/ApplicationQueryServiceTest.java

```diff
@@ -216,22 +216,25 @@ class Describe_findForKanban {
 		@Nested
 		class Context_with_user_id {
 
-			@DisplayName("지원서들의 현재 전형 정보를 반환한다")
+			@DisplayName("모든 전형 지원서 목록의 현재 전형 정보를 반환한다")
```

Comment:
```suggestion
			@DisplayName("현재 전형 정보들을 ProcessType별로 나누어 반환한다")
```
DisplayName을 이렇게 변경하는건 어떨까요?